alchemy/test/neon/branch.test.ts

```diff
@@ -0,0 +1,122 @@
+import { describe, expect } from "vitest";
+import { alchemy } from "../../src/alchemy.ts";
+import { destroy } from "../../src/destroy.ts";
+import { createNeonApi } from "../../src/neon/api.ts";
+import { NeonBranch } from "../../src/neon/index.ts";
+import { NeonProject } from "../../src/neon/project.ts";
+import { BRANCH_PREFIX } from "../util.ts";
+import "../../src/test/vitest.ts";
+
+const api = createNeonApi();
+
+const test = alchemy.test(import.meta, {
+  prefix: BRANCH_PREFIX,
+});
+
+describe("NeonBranch Resource", () => {
+  const testId = `${BRANCH_PREFIX}-test-neon-branch`;
+
+  const generateBranchName = () => `Test Branch ${testId}-${Date.now()}`;
```

Comment:
Do not use `Date.now()` suffix since that can leave dangling resources. Resource IDs in tests must be: 1) deterministic based on BRANCH_PREFIX, 2) unique across all tests and test suites

Reply:
Make sure to check the other test suites you just added